assign2/src/components/Fourthassign.jsx

```diff
@@ -0,0 +1,108 @@
+import React,{  useState } from 'react'
+import { Rating } from 'react-simple-star-rating'
+
+
+function Product(link,color){
+
+    const [rating, setRating] = useState(0) // initial rating value
+  
+  
+    const handleRating = (rate) => {
+      setRating(rate)
+    
+    }
+
+    return(
+        <div className='body-Border'>
+        <div className='bodyRating'>
+<div class="flex-container">
+<div >
+    <img  src={link}/>
+    <div >
+    <h1 className='product-Name'>Apple iPhone 12 (64GB) - {color}</h1> 
+    </div>  
+</div>
+<div className='tableMargine'>
+<table className="table">
+<tr>
+    <th table-head-spacing>Display:</th>
+    <td>6.1-inch (15.5 cm diagonal) Super Retina XDR display with True Tone</td>
+</tr>
+<tr>
+    <th table-head-spacing>Capacity:</th>
+    <td>4GB,128GB,256GB</td>
+</tr>
+<tr>
+    <th table-head-spacing>Splash, Water, and Dust Resistant:</th>
+    <td>All-glass and surgical-grade stainless steel design, water and dust resistant (rated IP68)</td>
+</tr>
+<tr>
+    <th table-head-spacing>Camera and Video:</th>
+    <td>Dual 12MP cameras with Portrait mode, Depth Control, Portrait Lighting, Smart HDR, and 4K Dolby Vision HDR video up to 60 fps</td>
+</tr>
+<tr>
+    <th table-head-spacing>Front Camera:</th>
+    <td>12MP TrueDepth front camera with Portrait mode, Depth Control, Portrait Lighting, and Smart HDR 3</td>
+</tr>
+<tr>
+    <th table-head-spacing>Power and Battery:</th>
+    <td>Video playback:Up to 17 hours, Video playback (streamed):Up to 11 hours, Audio playback:Up to 65 hours, 20W adapter or higher (sold separately), Fast-charge capable: Up to 50% charge in around 30 minutes with 20W adapter or higher</td>
+</tr>
+<tr>
+    <th table-head-spacing>In the Box:</th>
+    <td>iPhone with iOS 14, USB-C to Lightning Cable, Documentation</td>
+</tr>
+</table>
+
+<table class="table">
+<tr>
+    <th className='table-head-spacing'>About this item:</th>
+    <td>
+        <ul>
+            <li>6.1-inch (15.5 cm diagonal) Super Retina XDR display</li>
+            <li>Ceramic Shield, tougher than any smartphone glass</li>
+            <li>A14 Bionic chip, the fastest chip ever in a smartphone</li>
+            <li>Advanced dual-camera system with 12MP Ultra Wide and Wide cameras; Night mode, Deep Fusion, Smart HDR 3, 4K Dolby Vision HDR recording</li>
+            <li>12MP TrueDepth front camera with Night mode, 4K Dolby Vision HDR recording</li>
+            <li>Supports MagSafe accessories for easy attachment and faster wireless charging</li>
+            <li>iOS with redesigned widgets on the Home screen, all-new App Library, App Clips and more</li>
+        </ul>
+    </td>
+</tr>
+</table>
+<div>
+    <h1>Give Rating</h1>
+
+    <div className='App'>
+        <Rating onClick={handleRating} ratingValue={rating}/>
```

Comment:
What does Rating component do? What UI will it give to user to give rating? Could you please share a screenshot to get better idea?

Reply:
<img width="418" alt="Screenshot 2022-11-02 155159" src="https://user-images.githubusercontent.com/111748592/199465552-f8867adf-7354-4d51-a3ee-781b15d4b915.png">

it will show empty stars and onclick any star it will give rating accordingly.